### PR TITLE
UHM-7497 - Add Locations for MCOE

### PR DESCRIPTION
**Pull request by mseaton**

### CODE REVIEW

File: configuration/pih/pih-config-sierraLeone-kgh-test.json [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
     "Login Location": ["KGH Triage", "KGH MCH", "KGH NCD", "KGH Mental Health" ],
-    "Program Location": [ "KGH MCH", "KGH NCD", "KGH Mental Health" ]
+    "Program Location": [ "KGH MCH", "KGH NCD", "KGH Mental Health" ],
+    "Order Pathology Location": ["Labour and Delivery", "MCCU", "NICU", "Obstetric Post-Op", "Obstetric Surgery"],
+    "Inpatients App Location": ["Labour and Delivery", "MCCU", "NICU", "Obstetric Post-Op", "Obstetric Surgery"],
+    "Admission Note Location": ["KGH Triage", "Labour and Delivery", "MCCU", "NICU", "Obstetric Post-Op", "Obstetric Surgery"],
+    "Appointment Location": ["KGH Triage", "Labour and Delivery", "MCCU", "NICU", "Obstetric Post-Op", "Obstetric Surgery"],
+    "Admission Location": ["KGH Triage", "Labour and Delivery", "MCCU", "NICU", "Obstetric Post-Op", "Obstetric Surgery"],
+    "Transfer Location": ["Labour and Delivery", "MCCU", "NICU", "Obstetric Post-Op", "Obstetric Surgery"],
+    "Queue Location": ["KGH Triage", "Labour and Delivery", "MCCU", "NICU", "Obstetric Post-Op", "Obstetric Surgery"]
```
Comment (mogoodrich) [inline configuration/pih/pih-config-sierraLeone-kgh-test.json:11]: why did you move these here?  Our protocol is generally to use the csv files to set up locations, except in the case where a login tag needs to change based on server?

If there's a reason for this I'm missing, I'm fine with this.

**Review by mogoodrich — APPROVED** [review]
I guess my thought would be to add these tags to the locations-site-kgh-test.csv instead of doing this via config, if that works.

For clearing out the existing commits, you do two commits:
* Once that sets the value of all the new tags to flag in the location.csv,
* and then after that runs through the two staging servers, remove those new tags from locations.csv entirely?

File: configuration/pih/pih-config-sierraLeone-kgh-test.json (hunk 1 above)
Comment (mseaton) [reply configuration/pih/pih-config-sierraLeone-kgh-test.json:13]: We discussed this, and agreed to leave this here, but remember to consolidate into the csv at the time of adding to production.
